app/src/sharedTest/java/org/oppia/app/topic/review/TopicReviewFragmentTest.kt

```diff
@@ -46,25 +45,25 @@ class TopicReviewFragmentTest {
   )
 
   @Test
-  fun testTopicReviewFragment_loadFragment_displayReviewSkills_isSuccessful() {
-    launchTopicActivityIntent(TEST_TOPIC_ID_0).use {
+  fun testTopicReviewFragment_loadFragment_displayReviewTopics_isSuccessful() {
+    launchTopicActivityIntent(FRACTIONS_TOPIC_ID).use {
       onView(
         Matchers.allOf(
           withText(TopicTab.getTabForPosition(3).name),
           ViewMatchers.isDescendantOfA(withId(R.id.topic_tabs_container))
```

Comment:
Please import ViewMatchers to have better clean code.

Reply:
Done.